### DISCUSSION TITLE
Default for PortalElement

### DISCUSSION THREAD

**Discussion by BrianHung** [glideapps/glide-data-grid#253] [Ideas]
`data-grid-overlay-editor` attach itself to `body` if no `portalElement` is given and wrap itself in a div with `position: fixed; left: 0; top: 0; z-index: 9999;`, instead of erroring out and not rendering.

https://github.com/glideapps/glide-data-grid/blob/e62d735c78d9057e31cd47d14ccd64536abf133b/packages/core/src/data-grid-overlay-editor/data-grid-overlay-editor.tsx#L167-L193

Inspiration taken from `radix/primitive`:

https://github.com/radix-ui/primitives/blob/4dd90f1d38482e66cf4fd2970885d99a8fb3391c/packages/react/portal/src/Portal.tsx#L22-L65

**Comment by jassmith** [comment]
I wonder if we can do this without breaking back compat

**Reply by BrianHung** [reply]
Seems to only be used twice within the codebase: https://github.com/glideapps/glide-data-grid/search?q=portal

https://github.com/glideapps/glide-data-grid/blob/e62d735c78d9057e31cd47d14ccd64536abf133b/packages/core/src/data-grid-overlay-editor/data-grid-overlay-editor.tsx#L168

https://github.com/glideapps/glide-data-grid/blob/e62d735c78d9057e31cd47d14ccd64536abf133b/packages/cells/src/cells/dropdown-cell.tsx#L51

Maybe
```
document.getElementById("portal") ?? document.body
```

can work?
